content/trusted-content/official-images/_index.md

```diff
@@ -0,0 +1,40 @@
+---
+description: Guidelines for Official Images on Docker Hub
+keywords: Docker, docker, registry, accounts, plans, Dockerfile, Docker Hub, docs,
+  official,image, documentation
+title: Docker Official Images
+aliases:
+- /docker-hub/official_repos/
+- /docker-hub/official_images/
+---
+
+The [Docker Official Images](https://hub.docker.com/search?q=&type=image&image_filter=official)
+are a curated set of Docker repositories hosted on Docker Hub.
+
+These images provide essential base repositories that serve as the starting
+point for the majority of users.
+
+These include operating systems such as
+[Ubuntu](https://hub.docker.com/_/ubuntu/) and
+[Alpine](https://hub.docker.com/_/alpine/), programming languages such as
+[Python](https://hub.docker.com/_/python) and
+[Node](https://hub.docker.com/_/node), and other essential tools such as
+[memcached](https://hub.docker.com/_/memcached) and
+[MySQL](https://hub.docker.com/_/mysql).
+
+The images are some of the [most secure images](https://www.docker.com/blog/enhancing-security-and-transparency-with-docker-official-images/)
+on Docker Hub. This is particularly important as Docker Official Images are
+some of the most popular on Docker Hub. Typically, Docker Official images have
+few or no vulnerabilities.
```

Comment:
I would suggest steering very clear of conflating CVEs that apply to packages in the images with the much scarier "vulnerabilities" -- maybe "packages with applicable CVEs" or something?